plugins/binanceExchange_ws_test.go

```diff
@@ -0,0 +1,71 @@
+package plugins
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/stellar/kelp/model"
+	"github.com/stretchr/testify/assert"
+)
+
+// import (
+// 	"fmt"
+// 	"log"
+// 	"math"
+// 	"strconv"
+// 	"testing"
+// 	"time"
+
+// 	"github.com/stretchr/testify/assert"
+
+// 	"github.com/stellar/kelp/api"
+// 	"github.com/stellar/kelp/model"
+// )
+
+var testOrderConstraintsBinanceWs = map[string]map[model.TradingPair]model.OrderConstraints{
+	"binance": {
+		*model.MakeTradingPair(model.XLM, model.USDT): *model.MakeOrderConstraints(4, 5, 0.1),
+		*model.MakeTradingPair(model.XLM, model.BTC):  *model.MakeOrderConstraints(8, 4, 1.0),
+	},
+	"kraken": {
```

Comment:
why do we have a reference to kraken in a test for binance exchange?
maybe we can make this map simpler by just being `map[model.TradingPair]model.OrderConstraints` instead of `map[string]map[model.TradingPair]model.OrderConstraints`?

I think this code is copied from ccxtExchange.go which is wrong. copying from krakenExchange.go is correct.